websites/P/Paramount+/presence.ts

```diff
@@ -0,0 +1,235 @@
+const presence = new Presence({
+  clientId: "821433038335377418"
+});
+(async function () {
```

Comment:
delete this async function, why do you need it tho? If you need to await strings, do it later in presence.on()